c_src/build.sh

```diff
@@ -13,8 +13,8 @@ ERTS_INCLUDE_DIR=${ERTS_INCLUDE_DIR:-${ERL_ROOT}/erts-$(erlang_eval 'erlang:syst
 
 CC=${CC:-cc}
 DEFAULT_CFLAGS="-O3 -march=native -mtune=native -ggdb -Wall -Wextra"
-CFLAGS="-fPIC -I${ERTS_INCLUDE_DIR} -I${ERL_INCLUDE_DIR} -std=gnu11 ${CFLAGS:-$DEFAULT_CFLAGS}"
-LDFLAGS="-L${ERL_LIB_DIR} -lei ${LDFLAGS:-}"
+CFLAGS="-fPIC `pkg-config --cflags glib-2.0` -I${ERTS_INCLUDE_DIR} -I${ERL_INCLUDE_DIR} -std=gnu11 ${CFLAGS:-$DEFAULT_CFLAGS}"
```

Comment:
Consider using the `$()` syntax instead of backticks for command substitution in shell scripts: http://mywiki.wooledge.org/BashFAQ/082